SharperUniverse/Core/BaseSharperSystem.cs

```diff
@@ -49,6 +49,14 @@ protected BaseSharperSystem(GameRunner game)
         /// <returns></returns>
         public abstract Task CycleUpdateAsync(int deltaMs);
 
+	    public Task RegisterComponentAsync(SharperEntity entity, BaseSharperComponent component)
```

Comment:
Make sure all the tests and the fluent builder are updated for this change, as well as the examples.